pydoctor/model.py

```diff
@@ -112,7 +153,7 @@ def css_class(self) -> str:
         """A short, lower case description for use as a CSS class in HTML."""
```

Comment:
This is outside the scope of this or I think

Reply:
Yes, it's not necessary to address it now, but it's something you could keep in mind when designing new code, to make sure there is a way it can be fixed later.

Reply:
I've moved it to the utils module